server/app.js

```diff
@@ -37,5 +42,16 @@ app.use((err, req, res) => {
   res.status(err.status || 500);
   res.render('error');
 });
+mongoose.connect(process.env.DB_HOST, {
```

Comment:
Here as well, empty lines for splitting code into logical blocks

Reply:
Done as suggested